distribution/archives/integ-test-zip/build.gradle

```diff
@@ -17,7 +19,7 @@ group = "org.elasticsearch.distribution.integ-test-zip"
 
 tasks.named("processTestResources").configure {
   inputs.properties(project(':distribution').restTestExpansions)
```

Comment:
I believe we can remove all these explicit input declarations as well now. Since we are passing the expansions map to `filter()` the copy spec should be tracking those items as an input for us. Might want to confirm though by changing a map value and confirming the task is no longer up-to-date.

Reply:
That doesn't play out in my testing of this. If I remove those lines, then the tasks are incorrectly considered `up-to-date` even when the expansions map changes. Or, at least, that's how it's looking in my runs.

Reply:
:shrug: fair enough, there's no real disadvantage to be explicit here so I'm good with this as-is.